bases/br_ibge_nomes_brasil/quantidade_municipio_nome_2010/table_config.yaml

```diff
@@ -1,82 +1,103 @@
-source_bucket_name: basedosdados-dev
-project_id_staging: basedosdados-dev
-project_id_prod: basedosdados-dev
-table_id: quantidade_municipio_nome_2010 # AUTO GENERATED
-dataset_id: br_ibge_nomes_brasil # AUTO GENERATED
 
-version: v2.0 # REQUIRED
+# Igual ao dataset.name mas como lower case.
+# Exemplos: br_ibge_populacao, br_inep_censo_escolar
+dataset_id: br_ibge_nomes_brasil
+
+table_id: quantidade_municipio_nome_2010
 
-last_updated: 2020-10-21 # AUTO GENERATED
+# Título da tabela.
+title: 'Quantidade de pessoas nascidas - 2010'
 
 # Descreva a tabela. Essas são as primeiras frases que um usuário vai ver.
 # Você não precisa ser muito conciso. Sinta-se a vontade para dar exemplos de
 # como usar os dados.
-# Se souber, liste também aplicações: pesquisa, apps, etc. que usem os dados.
-description: |  # REQUIRED
-    Dados de quantidade de pessoas nascidas até 31 de julho de 2010 por município-nome.
+# Se souber, liste também aplicações: pesquisa, apps, etc. que usem os dados.,
+description: "Dados de quantidade de pessoas nascidas até 31 de julho de 2010 por município-nome."
+
+# As máximas unidades espaciais que a tabela cobre.
+# Exemplo:
+#   - sa.br
+spatial_coverage:
+    - sa.br
+
+# Anos cobertos pela tabela.
+# Preencher como lista de intervalos.
+# Exemplos: ['1995(1)2019'], ['2002(2)2010', '2016', '2020'].
+temporal_coverage:
+    - '2010'
+
+# A unidade temporal com qual a tabela é atualizada.
+# Opções em 'https://basedosdados.org/api/3/action/bd_available_options'
+update_frequency: ten_years
 
-# Quem está completando esse arquivo config?
+# Nível de observação da tabela: o que representa cada linha.
+observation_level:
+    - entity: name
+      columns:
+          - nome
+    - country: br
```

Comment:
para id município, precisamos colocar esse `country` antes de `entity`?

Reply:
Sim! sempre que a variável for geográfica tem que colocar o country